_posts/2019-11-21-vscode-quarkus-1.2.0.adoc

```diff
@@ -0,0 +1,172 @@
+* link:#language-feature-support-for-logging-level-values[Language feature support for logging level values]
+* link:#documentation-for-default-profiles[Documentation for default profiles]
+* link:#improved-input-validation-when-generating-a-new-quarkus-project[Improved input validation when generating a new Quarkus project]
+
+== Gradle support
+So far, Quarkus Tools for Visual Studio Code was geared towards Maven Quarkus
+projects. This release finally brings support to Gradle Quarkus projects.
+Moving forward, new features will be implemented with both Maven and Gradle in mind.
+
+=== Gradle - Generate new Quarkus project
+The project generation wizard now provides the option to choose between creating
+a Maven or a Gradle project. The rest of the wizard is the same as before.
+After going through the wizard, your new Maven or Gradle project will download
+and open automatically.
+
+To open the project generation wizard, open the VS Code command
+palette and call the `Quarkus: Generate a Quarkus project` command.
+
+image::01_gradle_project_gen.gif[alt="Generating a Gradle project"]
+
+=== Gradle - Debug Quarkus project
+The debug command will now detect whether your currently opened Quarkus
+project is a Maven or Gradle project, and will start the application by
+running the Quarkus Dev command
+(`mvn quarkus:dev` for Maven, `gradle quarkusDev` for Gradle).
+Once the application is running, the debugger will attach.
+
+To run the debug command, open the VS Code command palette and call
+the `Quarkus: Debug current Quarkus project` command.
+
+image::02_gradle_debug.gif[alt="Debugging a Gradle project"]
+
+=== Gradle - Add extensions to a Quarkus project
+The add extensions wizard now supports adding Quarkus extensions to Gradle projects. 
+To view the list of extensions to add, open the VS Code command palette and call the
+`Quarkus: Add extensions to current project` command.
+
+Here, the Eclipse Vert.x extension was added:
+
+image::03_gradle_add.gif[alt="Adding an extension to Gradle project"]
+
+Since Eclipse Vert.x was selected, the
+`./gradlew addExtension --extensions="quarkus-vertx"` command will
+run in the integrated terminal in order to add the extension:
+
+image::04_gradle_add.png[alt="Adding an extension to Gradle project"]
+
+In short, the project generation wizard can now generate Gradle projects,
+and the `Quarkus: Add extensions to current project` and
+`Quarkus: Debug current Quarkus project` commands work for
+_both_ Gradle and Maven projects.
+
+== New `application.properties` features
+
+=== Quick Fix for unknown property name
+The validation support provided by Quarkus Tools for Visual Studio Code
+checked for unknown property keys in your `application.properties` file.
+This release brings a new Quick Fix that suggests known properties for your
+unknown properties.
+This feature is kind of like autocorrect or "spellcheck" for your property keys.
```

Comment:
```suggestion
This feature is kind of like autocorrect or "spellcheck" for your properties keys.
```
Or `property key` maybe.

Reply:
Thank you for the suggestion, but I think "property keys" makes the most sense here.